pyanaconda/core/product.py

```diff
@@ -80,7 +80,18 @@ def get_product_values():
     # .buildstamp, environment, stupid last ditch hardcoded defaults.
     config = configparser.ConfigParser()
     config.add_section("Main")
-    config.set("Main", "IsFinal", os.environ.get("ANACONDA_ISFINAL", "false"))
+
+    # Get IsFinal property from /etc/os-release file
+    with open('/etc/os-release') as file:
+        for line in file:
+            key, value = line.split('=')
+            value = value.strip()
+            if key == 'RELEASE_TYPE':
+                if value in {'release', 'stable'}:
```

Comment:
nicer written as:
``config.set("Main", "IsFinal", str(value in {'release', 'stable'}))``

I think you can also remove  the setting of this varible in `data/liveinst/liveinst`, please mention the removal in separate commit. Users should rely on pyanaconda/core/product for reading this value.

@M4rtinK are you aware of any users of this env variable?

Reply:
Oh now I see the discussion above. Anyway, I believe this env variable should be removed, or set via this product.py file.

os.environ["ANACONDA_ISFINAL"] =   str(value in {'release', 'stable'})

Reply:
Done, and thanks for the suggestion how to do it more optimal way.